### ISSUE TITLE
Dashboard as collection home.

### ISSUE THREAD

**Issue by subzet** [metabase/metabase#46638]
**Is your feature request related to a problem? Please describe.**
Hey! It would be awesome to add dashboards to the collection home. It would be the same feature available now to make a dashboard the Metabase home, but for specific collections.

**Describe the solution you'd like**
Add a dashboard as the collection home.

**Describe alternatives you've considered**
-

**How important is this feature to you?**
We can live without it, but it would help us organize our official data collection for users.

**Comment by brunobergher** [comment]
@subzet Do you mean to configure a dashboard to be shown by default when a user goes to a collection, instead of the list of items? Mind sharing what you're trying to accomplish there? And how would you imagine users would go to the collection itself?

It's worth sharing that something we're shipping in the next release is the ability (also the default) to save questions _in_ dashboards, instead of in collections and then be referenced in the dashboards. That should turn many collections which today have 1 dashboard and many questions into just 1 dashboard, which might address your need.

**Comment by subzet** [comment]
Hey @brunobergher ! That's a great feature. What I mean is to be able to set a specific dashboard as the default view when you enter a collection. It is now posible to add a dashboard as the default metabase home, I'd like the same but when entering a collection, instead of seeing the list of questions stored in the collection.

**Comment by brunobergher** [comment]
Yup, I think I understand it. But how do you see users then getting to the collection itself? And what are you ultimately trying to accomplish? Something like creating a curated index for that collection, through which people would navigate?

**Comment by brunobergher** [comment]
Got it! Not sure we'll do this exactly, but passing on the feedback to the team.
